bps/bps_ApPipe.yaml

```diff
@@ -10,41 +10,37 @@ campaign: smallrun_example
 
 submitPath: ${PWD}/bps/{outCollection}
 computeSite: ncsapool
-# Memory allocated for each quantum, in MBs
+# The default BPS walltime is on the order of 3 days.
```

Comment:
Suggest empty lines before and after this block so it's clearer what the commented text is referring to.